init.lua

```diff
@@ -5,17 +5,25 @@ if not vim.tbl_contains(vim.opt.rtp:get(), base_dir) then
   vim.opt.rtp:append(base_dir)
 end
 
-require("lvim.bootstrap"):init(base_dir)
+require "lvim.utils.headless_fix"
 
-require("lvim.config"):load()
+-- This is only defined when updating using `lvim --update`
+---@diagnostic disable-next-line: undefined-field
+local updating = _G.__lvim_updating or false
 
-local plugins = require "lvim.plugins"
-require("lvim.plugin-loader").load { plugins, lvim.plugins }
+require("lvim.bootstrap"):init(base_dir, updating)
 
-local Log = require "lvim.core.log"
-Log:debug "Starting LunarVim"
+if not updating then
```

Comment:
```suggestion
```

It shouldn't be necessary to do this, since `updater.init()` should always quit at the end. We should even go a step further and force it to quit on a timer, just so it doesn't get stuck in the CI.

Reply:
As I mentioned previously, this is incorrect, the actions are asynchronous and handled in a callback tied to an even in Packer.

Reply:
Yup, fixed here: https://github.com/LunarVim/LunarVim/pull/2132/files#diff-8a5cb93842556000a3a4432fecd8e2e1c6d30fe458092fbb252494d5c94727a9R11

Reply:
Returning early works, but personally I think that makes it less clear. We can go with that if you think it's better.